ports/glaze/vcpkg.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "glaze",
+  "version": "1.2.0",
```

```suggestion
  "version": "1.2.1",
```
https://github.com/stephenberry/glaze/releases/tag/v1.2.1